test/Elastic.Apm.Tests.Utilities/README.md

```diff
@@ -0,0 +1,15 @@
+# Elastic.Apm.Tests.Utilities
+
+This project contains test specifications in JSON format that are used as test inputs in
```

Comment:
Since this is in the root of the `Elastic.Apm.Tests.Utilities` could we maybe add 1 general sentence before we go into the JSON specs here?

Thinking behind this is that if people open the folder and look at this readme they may get the impression this project is primarily about the JSON files.

So I'd start with something like this:

```
This project contains reusable utilities (e.g. in-memory or mock implementation of agent components,  test inputs, etc.) which are used in other test projects.
```

Reply:
Yes, that makes sense. I'll update the PR.